contracts/WitnetRequestsBoard.sol

```diff
@@ -274,6 +276,10 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
       _epoch,
       _index,
       requests[_id].drOutputHash), "Invalid PoI");
+    address payable relayer = payable(blockRelay.readRelayerAddress(_blockHash, _epoch));
+    uint256 relayerPayment = requests[_id].blockReward/2;
```

Comment:
Are relay DR inclusion and block similar in terms of gas cost?
Just asking myself if the factor 2 makes sense or could be improved...

Reply:
In theory they are both block inclusions (one for DR and the other one for result), so they should be yes.